### PR TITLE
zabbix_host ipmi_*-tests

### PR DESCRIPTION
**Pull request by eikef**
##### SUMMARY

This adds a couple of integration tests for the newly added ipmi_*-options introduced in #33435.

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
zabbix_host

##### ANSIBLE VERSION
2.5.0

### CODE REVIEW

File: test/integration/targets/zabbix_host/tasks/zabbix_host_tests.yml [hunk 1]
```diff
@@ -631,6 +631,97 @@
...
+    ipmi_username: username
+    ipmi_password: password
+  register: zabbix_host1
+
+- name: expect to succeed and that things have not changed
+  assert:
+      that:
+          - "zabbix_host1 is not changed"
+
+- name: expect to succeed and that things have not changed
```
Comment (D3DeFi) [inline test/integration/targets/zabbix_host/tasks/zabbix_host_tests.yml]: Duplicate with code on line 665?

Comment (eikef) [reply test/integration/targets/zabbix_host/tasks/zabbix_host_tests.yml]: Yepp, thanks for spotting that as well :-)

**Comment by eikef** [comment]
shipit

**Review by D3DeFi — APPROVED** [review]

**Comment by D3DeFi** [comment]
shipit

**Comment by eikef** [comment]
bot_status

**Comment by eikef** [comment]
shipit

**Comment by eikef** [comment]
bot_status

**Review by gundalow — APPROVED** [review]

**Pull request merged**